app/controllers/rents_controller.rb

```diff
@@ -19,4 +19,10 @@ def create_params
   def index
     render_paginated current_user.rents
   end
+
+  def show
+    @rent = Rent.find(params[:rent_id])
+    authorize @rent
+    render_paginated json: @rent
```

Comment:
It's a single record, we don't need to paginate the response, with a simple `render json: @rent` is enough